server/src/main/java/org/candlepin/model/GuestId.java

```diff
@@ -163,10 +166,12 @@ public boolean equals(Object other) {
         if (!(other instanceof GuestId)) {
             return false;
         }
+
         GuestId that = (GuestId) other;
-        if (this.getGuestId().equalsIgnoreCase(that.getGuestId()) &&
-            this.getAttributes().equals(that.getAttributes())) {
-            return true;
+        if (this.getGuestId().equalsIgnoreCase(that.getGuestId())) {
+            EqualsBuilder builder = new EqualsBuilder()
```

Comment:
This is just a map of strings, and the EqualsBuilder is just calling into .equals as the replaced code was doing. We're not gaining anything with this change.

Reply:
done